spec/lib/solidus_subscriptions/churn_buster_spec.rb

```diff
@@ -40,4 +40,18 @@
       expect(response).to be_success
     end
   end
+
+  describe '#report_payment_method_change' do
+    it 'reports the payment method change to Churn Buster' do
+      client = described_class.new(
+        account_id: '88416f1d-1984-4638-a47e-db3d5432f5c5',
+        api_key: 'e8e25da7982c5823359d54c121bea786408e5f8b8cce559484b8346a4dce6568',
```

Comment:
can we make these keys public? Also, I can't see them in the recorded cassette, is that expected?